src/core/meshcop/dataset.cpp

```diff
@@ -433,11 +433,8 @@ ThreadError Dataset::Restore(void)
 
     error = otPlatSettingsGet(mInstance, static_cast<uint16_t>(mType == Tlv::kActiveTimestamp ? kKeyActiveDataset :
```

Comment:
Don't we worry about other data, besides the timestamp, being still saved in settings?